spec/Powertrain/FuelSystem.vspec

```diff
@@ -123,3 +123,42 @@ IsFuelPortFlapOpen:
   datatype: boolean
   type: actuator
   description: Status of the fuel port flap(s). True if at least one is open.
+
+InstantaneousFuelEconomy:
+  datatype: float
+  type: sensor
+  unit: km/l
+  description: Real-time instantaneous fuel economy calculated over a short time window.
+  comment: This value represents the current fuel efficiency at any given moment during vehicle operation.
+           It is calculated based on the distance traveled and fuel consumed over a brief sampling period (typically around 500ms).
+           The calculation compares the vehicle state before and after the sampling window to determine instantaneous efficiency.
+           This metric refreshes frequently and can fluctuate significantly based on driving conditions, acceleration, and terrain.
+           Unlike average fuel economy measurements, this provides immediate feedback on current driving efficiency.
+
+CumulativeFuelEconomy:
```

Comment:
A bit similar idea as `AverageConsumption` but I like this name better so lets keep it